Okta.AspNet.Abstractions/OktaWebOptions.cs

```diff
@@ -13,10 +13,28 @@ public class OktaWebOptions
 
         public static readonly TimeSpan DefaultClockSkew = TimeSpan.FromMinutes(2);
 
+        /// <summary>
+        /// Gets or sets the Okta domain, i.e https://dev-123456.oktapreview.com.
+        /// </summary>
+        /// <value>
+        /// The Okta domain.
+        /// </value>
         public string OktaDomain { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Okta Authorization Server to use. The default value is default.
```

Comment:
tiny nitpick: `The default value is <c>default</c>.`
(I believe that adds `code formatting`)

Reply:
or even `"default"` in quotes works maybe